### PR TITLE
docs: improve document for setChecked(), setSelected() and setValue()

### PR DESCRIPTION
**Pull request by 38elements**
This is related to #700 and #729.

### CODE REVIEW

File: docs/api/wrapper/setChecked.md [hunk 1]
```diff
@@ -1,9 +1,19 @@
-## setChecked(value)
+## setChecked(checked)
 
-Sets the value of a radio or checkbox `<input>`.
+Sets the value to an input element of type checkbox or an input element of type radio.
```
Comment (eddyerburgh) [inline docs/api/wrapper/setChecked.md]: I think we should change this to mention the v-model:

`Sets checked value for input element of type checkbox or radio and updates `v-model` bound data.`

File: docs/api/wrapper/setChecked.md [hunk 2]
```diff
@@ -1,9 +1,19 @@
...
+
+When you try to set the value to state via `v-model` by `radioInput.element.checked = true; radioInput.trigger('input')`, `v-model` is not triggered. `v-model` is triggered by `change` event.
```
Comment (eddyerburgh) [inline docs/api/wrapper/setChecked.md]: I think we should add this to a `notes` sections after the arguments and example, I don't think most users need this info

File: docs/api/wrapper/setSelected.md [hunk 3]
```diff
@@ -1,9 +1,15 @@
-## setSelected(value)
+## setSelected()
 
-Sets a specified `<option>` as selected in a `<select>`.
+Selects a option element.
 
-- **Arguments:**
-  - `{Boolean} selected`
+When you try to set the value to state via `v-model` by `option.element.selected = true; parentSelect.trigger('input')`, `v-model` is not triggered. `v-model` is triggered by `change` event.
```
Comment (eddyerburgh) [inline docs/api/wrapper/setSelected.md]: Can you also move this to a `- **Notes:**` section

File: docs/api/wrapper/setValue.md [hunk 4]
```diff
@@ -1,6 +1,13 @@
 ## setValue(value)
 
-Sets the value of a text `<input>`.
+Sets the value to an input element of type text.
+
+`textInput.setValue(value)` is an alias of the following code.
```
Comment (eddyerburgh) [inline docs/api/wrapper/setValue.md]: Can you also move this to notes

File: docs/api/wrapper/setValue.md [hunk 5]
```diff
@@ -1,6 +1,13 @@
 ## setValue(value)
 
-Sets the value of a text `<input>`.
+Sets the value to an input element of type text.
```
Comment (eddyerburgh) [inline docs/api/wrapper/setValue.md]: Sets the value to an input element of type text.

`Sets value of a text-control input element and updates `v-model` bound data`

File: docs/api/wrapper/setSelected.md [hunk 6]
```diff
@@ -1,9 +1,15 @@
-## setSelected(value)
+## setSelected()
 
-Sets a specified `<option>` as selected in a `<select>`.
+Selects a option element.
```
Comment (eddyerburgh) [inline docs/api/wrapper/setSelected.md]: Selects a option element.

`Selects a option element and updates v-model bound data.`

**Comment by 38elements** [comment]
Thank you for pointing out.
I changed them.

File: docs/api/wrapper/setSelected.md [hunk 7]
```diff
@@ -1,9 +1,6 @@
-## setSelected(value)
+## setSelected()
 
-Sets a specified `<option>` as selected in a `<select>`.
-
-- **Arguments:**
-  - `{Boolean} selected`
+Selects a option element and updates `v-model` bound data.
```
Comment (lmiller1990) [inline docs/api/wrapper/setSelected.md]: Selects an option

**Review by lmiller1990 — APPROVED** [review]
nice job! Just one small comment, a -> an.

File: docs/api/wrapper/setSelected.md (hunk 7 above)
Comment (38elements) [reply docs/api/wrapper/setSelected.md]: Thanks.

**Review by eddyerburgh — APPROVED** [review]
Thanks

**Pull request merged**